package.json

```diff
@@ -17,7 +17,7 @@
     "test-coverage": "nyc npm test",
     "report-coverage": "codecov < ./coverage/lcov.info",
     "api": "cross-env ENV_CONFIG=production node src/runner.js",
-    "postinstall": "babel src --out-dir dist"
+    "prepublish": "npm run build"
```

Comment:
I would like to remind that in case when we have ```prepublish``` npm task, this will be run every time when we use ```npm install```